bindings/pydeck/pydeck/io/templates/index.j2

```diff
@@ -5,7 +5,9 @@
 {% if google_maps_key %}
     <script src="https://maps.googleapis.com/maps/api/js?key={{google_maps_key}}&libraries=places"></script>
 {% endif %}
+{% if mapbox_key %}
```

Comment:
- latest Mapbox version is 1.13.0
- Does this mean that you can't use the Carto map styles (no key required)?

Reply:
Defaults to Carto if no key is set.
I'll update the version.

Reply:
I think the point was: are you still including the `mapbox-gl.js` script when a mapbox key is not set?